.gitlab-ci.yml

```diff
@@ -173,7 +173,7 @@ windows:
     - stable
     - triggers
   script:
-    - sh scripts/gitlab-build.sh x86_64-pc-windows-msvc x86_64-pc-windows-msvc installer "" "" windows
+    - sh scripts/gitlab-build.sh x86_64-pc-windows-msvc x86_64-pc-windows-msvc "" "" "" windows
```

Comment:
Not sure what (if anything) should be for the $ARC param instead of installer

Reply:
$ARC is just a string that goes into the binary name. I'm pretty sure it's unused after this PR, however, for consistency reasons, could you change it to `amd64`?